File: PioneersWeb/src/TransportBundle/Entity/Bus.php
<?php

namespace TransportBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Bus
 *
 * @ORM\Table(name="bus", uniqueConstraints={@ORM\UniqueConstraint(name="name", columns={"name"})})
 * @ORM\Entity(repositoryClass="TransportBundle\Repository\BusRepository")
 */
class Bus
{


    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }



    /**
     * @param int $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return int
     */
    public function getCapacite()
    {
        return $this->capacite;
    }

    /**
     * @param int $capacite
     */
    public function setCapacite($capacite)
    {
        $this->capacite = $capacite;
    }

    /**
     * @return string
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * @param string $name
     */
    public function setName($name)
    {
        $this->name = $name;
    }

    /**
     * @return string
     */
    public function getChauffeur()
    {
        return $this->chauffeur;
    }

    /**
     * @param string $chauffeur
     */
    public function setChauffeur($chauffeur)
    {
        $this->chauffeur = $chauffeur;
    }
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var integer
     *
     * @ORM\Column(name="Capacite", type="integer", nullable=false)
     */
    private $capacite;

    /**
     * @var string
     *
     * @ORM\Column(name="name", type="string", length=30, nullable=false)
     */
    private $name;

    /**
     * @var string
     *
     * @ORM\Column(name="Chauffeur", type="string", length=30, nullable=false)
     */
    private $chauffeur;


}

File: PioneersMobile/src/Entity/Bus.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entity;

/**
 *
 * @author ASUS-PC
 */
public class Bus {
    
        private int id,Capacite;
    private String name,Chauffeur;

    public Bus() {
    }


    

    @Override
    public String toString() {
        return "Bus{" + "id=" + id + ", Capacite=" + Capacite + ", name=" + name + ", Chauffeur=" + Chauffeur + '}';
    }
    
    
    public Bus(int id,  String name,int Capacite, String Chauffeur) {
        this.id = id;
        this.Capacite = Capacite;
        this.name = name;
        this.Chauffeur = Chauffeur;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getCapacite() {
        return Capacite;
    }

    public void setCapacite(int Capacite) {
        this.Capacite = Capacite;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getChauffeur() {
        return Chauffeur;
    }

    public void setChauffeur(String Chauffeur) {
        this.Chauffeur = Chauffeur;
    }
    
    
}
